src/test_vectors/bip340_vector.rs

```diff
@@ -0,0 +1,41 @@
+// This is the first row (index = 0) from https://github.com/bitcoin/bips/blob/master/bip-0340/test-vectors.csv
```

Comment:
check option to generate directly from [bip340_reference.py](https://github.com/QED-it/zcash-test-vectors/pull/12/files#diff-1f4541b74bf12531b8d5a46e5469902e5b77cca66d770911691ce39b1d1bf056)